docs/samples/batcher/basic/README.md

```diff
@@ -0,0 +1,66 @@
+# Inference Batcher Demo
+
+We first create a sklearn predictor with a batcher. The "maxLatency" is set to a big value (5000.0 milliseconds) to make us be able to observe the batching process.
+
+```
+apiVersion: "serving.kubeflow.org/v1alpha2"
+kind: "InferenceService"
+metadata:
+  name: "sklearn-iris"
+spec:
+  default:
+    predictor:
+      minReplicas: 1
+      batcher:
+        maxBatchSize: "32"
+        maxLatency: "5000.0"
+        timeout: "60"
+      sklearn:
+        storageUri: "gs://kfserving-samples/models/sklearn/iris"
+```
+
+Let's apply this yaml:
+
+```
+kubectl create -f sklearn-batcher.yaml
+```
+
+We can now send 2 requests to the sklearn model under 2 ssh terminal tabs. Use `kfserving-ingressgateway` as your `INGRESS_GATEWAY` if you are deploying KFServing as part of Kubeflow install, and not independently.
+
+```
+MODEL_NAME=sklearn-iris
+INPUT_PATH=@./iris-input.json
+INGRESS_GATEWAY=istio-ingressgateway
+CLUSTER_IP=$(kubectl -n istio-system get service $INGRESS_GATEWAY -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+SERVICE_HOSTNAME=$(kubectl get inferenceservice sklearn-iris -o jsonpath='{.status.url}' | cut -d "/" -f 3)
+curl -v -H "Host: ${SERVICE_HOSTNAME}" http://$CLUSTER_IP/v1/models/$MODEL_NAME:predict -d $INPUT_PATH -H "Content-Type: application/json"
```

Comment:
We can use `vegeta` or `hey` to send multiple requests at same time, so it is a better demo for batching.